Sources/Views/Badges/BadgeView.swift

```diff
@@ -26,9 +19,25 @@ open class BadgeView: BadgeContainerView {
         imageView.contentMode = .scaleAspectFit
         return imageView
     }()
+
+    /// The constraint between the image view's width and the
+    /// container view's width. Can be adjusted to change how much
+    /// of the container the image fills
+    private var imageWidthConstraint: NSLayoutConstraint?
```

Comment:
As it's done in `setup()` (inside `init`) I think `NSLayoutConstraint!` is OK because it should be set by the end of initializing. I guess similarly to an outlet.